### DISCUSSION TITLE
File Harvesting is not working as expected

### DISCUSSION THREAD

**Discussion by gkrishna240988** [wixtoolset/issues#8184] [Questions]
When we try to exclude files in the Files section, the build fails with below error. We followed the documentation from #[here](https://wixtoolset.org/docs/fivefour/#file-harvesting)
**Error:**
wix.exe : error WIX0001: System.IO.IOException: The filename, directory name, or volume label syntax is incorrect.

**Comment by Thylon1** [comment]
Could you provide a code example to reproduce the error or even better a repro repo?
